internal/provider/models/alerts/absence.go

```diff
@@ -0,0 +1,178 @@
+package alerts
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	. "github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	. "github.com/mezmo/terraform-provider-mezmo/internal/client"
+	. "github.com/mezmo/terraform-provider-mezmo/internal/provider/models/modelutils"
+)
+
+const ALERT_TYPE_ABSENCE = "absence"
+
+type AbsenceAlertModel struct {
+	Id                StringValue `tfsdk:"id"`
+	PipelineId        StringValue `tfsdk:"pipeline_id"`
+	ComponentKind     StringValue `tfsdk:"component_kind"`
+	ComponentId       StringValue `tfsdk:"component_id"`
+	Inputs            ListValue   `tfsdk:"inputs"`
+	Active            BoolValue   `tfsdk:"active"`
+	Name              StringValue `tfsdk:"name" user_config:"true"`
+	Description       StringValue `tfsdk:"description" user_config:"true"`
+	EventType         StringValue `tfsdk:"event_type"`
```

Comment:
Doesn't this attribute also need `user_config:"true"`?

Reply:
yes, good catch